### PR TITLE
ci: log result

### PR DESCRIPTION
**Pull request by enitrat**
Result is now displayed in the workflow summary section

## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Resolves: #

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: .github/workflows/gas_reports.yml [hunk 1]
```diff
@@ -35,3 +35,4 @@ jobs:
           result="${result//$'\n'/'%0A'}"
           result="${result//$'\r'/'%0D'}"
           echo "::set-output name=result::${result}"
```
Comment (ClementWalter) [inline .github/workflows/gas_reports.yml:37]: can remove this line I think (for setting the setp.output value, not used anymore, and generating a deprecation warning in the CI)

**Review by ClementWalter — CHANGES_REQUESTED** [review]
can we make the small modif required in the meantime to clear these warnings

[compare-snapshot](https://github.com/kkrt-labs/kakarot-ssj/actions/runs/6180880394/job/16778019564)
The following actions uses node12 which is deprecated and will be forced to run on node16: actions/checkout@v2, actions/setup-python@v2. For more info: https://github.blog/changelog/2023-06-13-github-actions-all-actions-will-run-on-node16-instead-of-node12-by-default/

[compare-snapshot](https://github.com/kkrt-labs/kakarot-ssj/actions/runs/6180880394/job/16778019564#step:5:15)
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

File: .github/workflows/gas_reports.yml (hunk 1 above)
Comment (enitrat) [reply .github/workflows/gas_reports.yml:37]: ah yes indeed. I though we still had the comment active for local branches. will change

File: .github/workflows/gas_reports.yml [hunk 2]
```diff
@@ -12,10 +12,10 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
```
Comment (ClementWalter) [inline .github/workflows/gas_reports.yml:18]: can't we use `v4` as in the `gas_snapshot` workflow?

**Review by ClementWalter — APPROVED** [review]

File: .github/workflows/gas_reports.yml (hunk 2 above)
Comment (enitrat) [reply .github/workflows/gas_reports.yml]: I updated it to v4

**Comment by enitrat** [comment]
@ClementWalter can we merge?

**Review by ClementWalter — APPROVED** [review]

**Pull request merged**